pkg/sql/importer/import_job.go

```diff
@@ -1461,6 +1463,11 @@ func (r *importResumer) dropTables(
 		return nil
 	}
 
+	// useDeleteRange indicates that the cluster has been finalized to 22.1 and
+	// can use MVCC compatible DeleteRange to with range tombstones during an import
+	// rollback.
+	useDeleteRange := r.settings.Version.IsActive(ctx, clusterversion.MVCCRangeTombstones)
```

Comment:
I think it might be prudent to have an internal cluster setting for this too, in case we find something horribly wrong.

@dt Any good reason why we wouldn't? Like, would it be helpful to have a guarantee that all operations are MVCC-compliant by 23.1?

Reply:
Seems reasonable to me; i'll defer to @dt here.

Reply:
As discussed in MVCC ops meeting, no cluster setting for now, but may add during stability.